docs/source/getting-started.md

```diff
@@ -205,26 +205,36 @@ as follows:
 c.JupyterHub.cookie_secret_file = '/srv/jupyterhub/cookie_secret'
 ```
 
-The content of this file should be a long random string. An example would be to generate this
-file as:
+The content of this file should be a long random string encoded in MIME Base64. An example would be to generate thisfile as:
 
 ```bash
-openssl rand -hex 1024 > /srv/jupyterhub/cookie_secret
+openssl rand -base64 2048 > /srv/jupyterhub/cookie_secret
```

Comment:
No, this is bash, not python, no need to import anything. 

Should we also check for whether the content of the file might be hex (set of chars in 0-9A-F?), in which case decoding it as base64 lower entropy. And if so print a warning ?